TgeeT/Questioner_v2
# Feature #163283772 user signup endpoint

#### What does this PR do?

This PR create an endpoint for users to provide their signup details which are stored in a Database
#### Description of Task to be completed?

- first was to ensure that the database configuration works and the users tables are created 
- write database models to store the data in the DB
- Write user views to retrieve the data from the user
- write validations for the data being input by the user 

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone this repository [here](https://github.com/TgeeT/Questioner_v2.git) 
2. Create a virtual environment and activate it.
3. git checkout to develop branch
4. Install requirements with 'pip install -r requirements.txt'
5. Run the application and with post man, post at the endpoint 'api/v2/user'

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/163283772

### Screenshots
###### postman
![image](https://user-images.githubusercontent.com/45405036/51493070-19cae180-1dc5-11e9-9fad-f6c232205d09.png)
